app/helpers/edition_url_helper.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module EditionUrlHelper
+  def edition_public_url(edition)
+    return unless edition.base_path
+
+    Plek.new.website_root + edition.base_path
+  end
+
+  def edition_secret_preview_url(edition)
```

Comment:
Since we don't have a non secret form of this I'd suggest the secret part is redundant. It might not be necessary to have the private method here too since it's a one liner.